rules/no-empty-file.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+const MESSAGE_ID = 'no-empty-file';
+const messages = {
+	[MESSAGE_ID]: 'Prefer `{{replacement}}` over `{{value}}`.',
+};
+
+/** @param {import('eslint').Rule.RuleContext} context */
+const create = () => ({
+	Program(node) {
+		const problem = {
+			node,
+			messageId: MESSAGE_ID,
+			data: {
+				value: 'unicorn',
+				replacement: '🦄',
+			},
+		};
+
+		if (node.body.length === 1 && node.body[0].directive === 'use strict') {
```

Comment:
Should use `node.body.every()` or `node.body.some()`, so `;;'use strict';` will consider empty.

Reply:
Excellent suggestion!